src/foam/dao/QueryCachingDAO.js

```diff
@@ -26,7 +26,10 @@ foam.CLASS({
     function init() {
       // if anything changes in the delegate -> clear cache
       // Can happen if the dao is modified outside the DAOController (for eg. in wizards)
-      this.onDetach(this.delegate.listen(this.FnSink.create({ fn: () => this.cmd_(x, foam.dao.DAO.PURGE_CMD) })));
+      this.onDetach(this.delegate.listen({ 
```

Comment:
Can you add:
// only call purge cache on put and remove events to avoid infinite loop when used alongside cachingdao

But what is there besides put and remove? reset? Wouldn't we want that to work? Shouldn't we just suppress feedback?